partiql-planner/src/testFixtures/kotlin/org/partiql/planner/test/TestId.kt

```diff
@@ -0,0 +1,39 @@
+package org.partiql.planner.test
+
+/**
+ * Represents a unique identifier for a test case.
+ *
+ * This is used to uniquely identify a test case within a test suite.
+ */
+class TestId private constructor(
+    private val _qualifier: Array<String>,
+    private val _name: String
+) {
+
+    private fun getQualifier(): Array<String> {
+        return _qualifier
+    }
+
+    private fun getName(): String {
+        return _name
+    }
```

Comment:
Are these getters necessary? Since this is just in the tests, this indirection is not needed. I feel like it would be much simpler to just directly reference the `_qualifer` and `_name`.